package.json

```diff
@@ -36,11 +36,10 @@
   },
   "devDependencies": {
     "chart.js": "2.8.0",
-    "pc-nrfconnect-devdep": "git+https://github.com/NordicSemiconductor/pc-nrfconnect-devdep.git#semver:^3.1.0",
-    "react-chartjs-2": "2.7.6",
-    "react-rangeslider": "2.2.0",
     "mathjs": "6.0.2",
-    "mousetrap": "1.6.3"
+    "mousetrap": "1.6.3",
+    "pc-nrfconnect-devdep": "git+https://github.com/NordicSemiconductor/pc-nrfconnect-devdep.git#semver:3.3.0",    "react-chartjs-2": "2.7.6",
```

Comment:
Here's a small formatting issue :)

Reply:
Darn. That happened to me in several files and I thought I had caught them all. Obviously not. I will never be a good Pokemon player! ;-) 

Fixed now, ready for review.